pkg/packagekit/package_wix.go

```diff
@@ -120,6 +120,11 @@ func PackageWixMSI(ctx context.Context, w io.Writer, po *PackageOptions, include
 			wix.ServiceArgs([]string{"svc", "-config", po.FlagFile}),
 			wix.ServiceDescription(fmt.Sprintf("The Kolide Launcher (%s)", po.Identifier)),
 		)
+
+		if po.DisableService {
+			wix.WithDisabledService()(launcherService)
```

Comment:
ha. This is, obviously, a fine way to use the functional arguments. But I don't think I've seen it before. Took me a couple minute to figure out how it worked. <3